src/Nethermind/Nethermind.EthereumTests.Benchmark/SingleTests.cs

```diff
@@ -22,30 +22,18 @@
 namespace Nethermind.EthereumTests.Benchmark
 {
     [SimpleJob(RunStrategy.ColdStart, warmupCount:1, targetCount: 5)]
-    public class CallsWithLargeData : GeneralStateTestBase
+    public class SingleTests : GeneralStateTestBase
     {
-        [Benchmark]
-        public void Gas10M_0xDead()
-        {
-            FileTestsSource source = new(@"AdditionalTests\CallsWithLargeData\10MGas_0xdead.json");
-            var tests = source.LoadGeneralStateTests();
-        
-            foreach (GeneralStateTest test in tests)
-            {
-                RunTest(test);
-            }
-        }
-        
-        [Benchmark]
-        public void Gas10M_BnAdd()
-        {
-            FileTestsSource source = new(@"AdditionalTests\CallsWithLargeData\10MGas_bnAdd.json");
-            var tests = source.LoadGeneralStateTests();
-        
-            foreach (GeneralStateTest test in tests)
-            {
-                RunTest(test);
-            }
-        }
+        // [Benchmark]
+        // public void TestName()
```

Comment:
remove commented out code

Reply:
I wanted somehow show that we can run a singular test so I left it as an example.
Would be better to use actual test.
Any ideas which test I could include here?